frontend/src/components/Home.jsx

```diff
@@ -5,22 +5,17 @@ import { connect } from "react-redux";
 // Local imports
 import { actions } from "../store";
 
-// *** issues *** environment variables are undefined
-// import dotenv from 'dotenv'
-// dotenv.config()
-
 window.fbAsyncInit = function() {
   FB.init({
-    appId: process.env.FB_APP_ID,
+    appId: "507675923424391",
```

Comment:
The changes to hardcode FB environment variables are already in master. Merging this will result to conflicts, and then we'll have to resolve master. Have you tried merging or rebasing master to your branch?